spot-contracts/contracts/RolloverVault.sol

```diff
@@ -872,6 +868,9 @@ contract RolloverVault is
             if (rolloverTokens[i] != underlying_) {
                 // sync deployed asset retrieved from perp
                 _syncDeployedAsset(tokenOutOfPerp);
+
+                // Clean up after rollover, merge seniors from perp with vault held juniors to recover more underlying.
+                _redeemTranche(ITranche(address(tokenOutOfPerp)));
```

Comment:
We could have _redeemTranche return a bool. True, means some were redeemed (and synced).

Then we could avoid the double sync that would happen because of line 870 right above.

Reply:
Rather, we can move `_syncDeployedAsset` into `_redeemTranche`

Reply:
Yep, that's a better solution 👍